trinity/extensibility/plugin.py

```diff
@@ -80,18 +118,25 @@ class BasePlugin(ABC):
     @abstractmethod
     def name(self) -> str:
         """
-        Describe the name of the plugin
+        Describe the name of the plugin.
         """
         raise NotImplementedError(
```

Comment:
you could replace this `NotImplementedError` with a `pass` (our current convention).